scripts/hooks/prepare-commit-msg

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+COMMIT_MSG_FILE=$1
+STATUS_GO_VERSION_FILE_IS_BEING_COMMITTED=$(git diff --cached --name-only -r | grep status-go-version.json)
+
+if [[ -z "${STATUS_GO_VERSION_FILE_IS_BEING_COMMITTED}" ]]; then
+  exit 0 # No status-go-version.json changes found.
+fi
+
+MERGE_BASE=$(git merge-base -a develop "$(git rev-parse --abbrev-ref HEAD)" develop)
+GO_COMMIT_MERGE_BASE=$(git show "$MERGE_BASE":status-go-version.json | jq -r '."commit-sha1"')
+GO_COMMIT_CURRENT=$(cat status-go-version.json | jq -r '."commit-sha1"')
+
+GITHUB_LINK="https://github.com/status-im/status-go/compare/$GO_COMMIT_MERGE_BASE...$GO_COMMIT_CURRENT"
+
+echo "$GITHUB_LINK"
+if [ -f "$COMMIT_MSG_FILE" ]; then
```

Comment:
If you're using bash you should consistently use double square brackets.

Reply:
Also, when referencing variables it's better to use the curly brackets format too.